python/xorq/vendor/ibis/tests/test_profile.py

```diff
@@ -58,7 +60,7 @@ def test_profiles(monkeypatch, tmp_path):
     assert default_profile_dir == pathlib.Path("~/.config/letsql/profiles").expanduser()
     profiles = Profiles()
     assert profiles.profile_dir == default_profile_dir
-    assert not profiles.list()
+    assert not profiles.list()  # why do this ?
```

Comment:
do we expect to have any profiles pre-populated?

Reply:
No I do not think we want any pre-populated, especially in CICD but if running tests locally I would rather just clean up.